src/main/java/com/facebook/presto/maven/ServiceDescriptorGenerator.java

```diff
@@ -119,7 +119,7 @@ private List<Class<?>> findImplementationsOf(Class<?> implementationTemplate, UR
         List<Class<?>> implementations = Lists.newArrayList();
         List<String> classes = FileUtils.getFileNames(classesDirectory, "**/*.class", null, false);
         for (String classPath : classes) {
-            String className = classPath.substring(0, classPath.length() - 6).replace('/', '.');
+            String className = classPath.substring(0, classPath.length() - 6).replace(File.separatorChar, '.');
```

Comment:
replaceAll?

Reply:
@wanglinsong No,  [replace](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#replace-char-char-) should be used, because it replaces **all** occurrences:
```
public String replace(char oldChar, char newChar)
Returns a string resulting from replacing all occurrences of oldChar in this string with newChar.
```
while [replaceAll](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#replaceAll-java.lang.String-java.lang.String-) works with regular expressions.

But I agree that it's confusing naming.